src/app/project-sets/page.tsx

```diff
@@ -0,0 +1,57 @@
+'use client'
+
+import React from "react"
+import {columns} from "@/app/project-sets/columns"
+import {DataTable} from "@/components/ui/data-table"
+import PageView from "@/components/views/Page"
+import {type ApiProjectSet} from "../../../types/api/teams"
+import {useRouter} from "next/navigation"
+import {type ProjectSet} from "../../../types/pages/projectSet"
+
+async function getProjectSetsData(): Promise<ProjectSet[]> {
+    const response = await fetch(process.env.DJANGO_BACKEND_URI + '/api/v1/teamset-templates')
+    if (!response.ok) {
+        throw new Error('Unable to fetch project sets from API.')
+    }
+    const teamSets = await response.json()
+    return teamSets.map(({id, name, teams}: ApiProjectSet) => ({
+        id,
+        name,
+        numProjects: teams.length,
+    } as ProjectSet))
+}
+
+function ProjectSetsPage() {
+    const router = useRouter()
+
+    const handleRowClick = (row: ProjectSet) => {
+        router.push(`/project/${row.id}`)
```

Comment:
Consider a project-set/id/project/id pattern here. This would allow for the breadcrumbs to lead you back to the project set easily 

There's other ways to do this but consider

Reply:
Makes sense